### DISCUSSION TITLE
Possible options for SSH integration on iOS

### DISCUSSION THREAD

**Discussion by cmyplay** [Vinzent03/obsidian-git#795] [Problems]
SSH is an essential tool for syncing with git servers that aren't GitHub.

It would be amazing if Obsidian Git could support [LibGit2](https://libgit2.org) (either through Javascript bindings like
[WASM-Git](https://github.com/petersalomonsen/wasm-git) or by adding a way to call commands in[ a-shell](https://github.com/holzschu/a-shell?tab=readme-ov-file) on iOS (which has LibGit2 preinstalled).

Thanks for the great work.

**Comment by Vinzent03** [comment]
Using wasm-git is probably still not possible, because of the async obsidian file system api. And I don't think it's possible to call commands to any shell from an Obsidian plugin, because the native access is quite restricted. So I currently see no way to support ssh on mobile or ios in specific.
